src/events/types.rs

```diff
@@ -83,10 +83,14 @@ pub trait EventType {
 }
 
 /// A lattice event
-#[derive(Debug)]
+// TODO:  this `Deserialize` impl _compiles_ but doesn't let you deserialize an event because of our custom Serialize
```

Comment:
@thomastaylor312 just a note, this is probably why tests fail and all. The notifications that we try to send to remove an event or register events fail to deserialize on the other end because they use the derived `Deserialize` but the manually implemented `Serialize`

Reply:
@thomastaylor312 is this still an issue?

Reply:
Nope, all fixed now, forgot to remove todo